lte/gateway/c/li_agent/PDUGenerator.h

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <tins/tins.h>
+#include <tins/pdu.h>
+
+#include "magma_logging.h"
+#include "DirectorydClient.h"
+#include "ProxyConnector.h"
+
+#define XID_LENGTH 16
+#define LI_X3_LINK_TYPE 0x08ae
+#define PDU_VERSION 2
+#define PDU_TYPE 2
+#define IP_PAYLOAD_FORMAT 5
+#define DIRECTION_TO_TARGET 2
+#define DIRECTION_FROM_TARGET 3
+
+struct pdu_info {
+  uint16_t version;
+  uint16_t pdu_type;
+  uint32_t header_length;
+  uint32_t payload_length;
+  uint16_t payload_format;
+  uint16_t payload_direction;
+  uint8_t xid[XID_LENGTH];
+};
+
+struct conditional_attributes {
+  uint64_t timestamp;
+};
+
+using namespace Tins;
```

Comment:
Can we import just the entities from Tins that we need? [You may not use a using-directive to make all names from a namespace available.](https://google.github.io/styleguide/cppguide.html#Namespaces)